src/assets/scripts/client/navigationLibrary/NavigationLibrary.js

```diff
@@ -316,4 +322,41 @@ export default class NavigationLibrary {
             flightPhase === FLIGHT_PHASE.TAXI ||
             flightPhase === FLIGHT_PHASE.WAITING;
     }
+
+    /**
+     * Check all fixes used in procedures, and gather a list of any fixes that are
+     * not defined in the `fixes` section of the airport file, then sort and print
+     * that list to the console.
+     *
+     * @for NavigationLibrary
+     * @method showConsoleWarningForUndefinedFixes
+     */
+    showConsoleWarningForUndefinedFixes() {
+        const allFixNames = this._getAllFixNames();
+        const missingFixes = allFixNames.filter((fix) => !FixCollection.findFixByName(fix));
+
+        if (missingFixes.length < 1) {
+            return;
+        }
+
+        console.warn(`The following fixes have yet to be defined in the "fixes" section: ${missingFixes}`);
+    }
+
+    /**
+     * Gathers a unique, sorted list of all fixes used in all known procedures
+     *
+     * @for NavigationLibrary
+     * @method _getAllFixNames
+     * @return {array<string>} ['fixxa', 'fixxb', 'fixxc', ...]
+     * @private
+     */
+    _getAllFixNames() {
+        const sidFixes = this.sidCollection.getAllFixNames();
```

Comment:
There are a few blocks below with this same structure, and my suggestion here will apply to those as well.

Instead of `_uniq(thing.concat(otherThing)).sort`, we can do this:
```javascript
const allFixNames = [
  ...this.sidCollection.getAllFixNames(),
  ...this.starCollection.getAllFixNames()
];
const uniqueFixNames = _uniq(allFixNames);
const allNonVectorFixes = _filter(uniqueFixNames, (fix) => fix.indexOf(VECTOR_WAYPOINT_PREFIX) === INVALID_INDEX);

return allNonVectorFixes.sort();
```
This does a few things for us:
- gets rid of `concat` which will mutate an existing array (not desirable).
- gives us an aptly named var `uniqueFixNames` that provides a little more context
- adds `_filter()` which, although there is a native version, we are using lodash all over the place and should use lodash methods when available.  Plus native `filter` isn't supported by some versions of IE (though this argument is not a valid today as it was a year or two ago)